File: TRNMNT/ClientApp/app/core/model/participant.model.ts
export class ParticipantModel {

    public participantId: AAGUID;
    public firstName: string;
    public lastName: string;
    public team: string;
    public dateOfBirth: Date;
    public isApproved: boolean;
    public categoryId: string;
    public weightDivisionId: string;
    public eventId: AAGUID;
    public userId: string;
    constructor() { }

}

File: TRNMNT.Core/Model/Participant/ParticipantModel.cs
using System;

namespace TRNMNT.Core.Model.Participant
{
    public class ParticipantModel
    {
        public Guid ParticipantId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string DateOfBirth { get; set; }
        public string WeightDivision { get; set; }
        public string Team { get; set; }
        public string Category { get; set; }
        public string IsMember { get; set; }
    }

}